workflows/pipe-common/shell/pipe_mail_enabler

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+# Copyright 2017-2020 EPAM Systems, Inc. (https://www.epam.com/)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+echo "Linking $COMMON_REPO_DIR_MUTUAL_LOC/shell/pipe_mail as default linux mail client"
+rm -rf "/usr/bin/mail" && ln -s "$COMMON_REPO_DIR_MUTUAL_LOC/shell/pipe_mail" "/usr/bin/mail" \
+    && rm -rf "/bin/mail" && ln -s "$COMMON_REPO_DIR_MUTUAL_LOC/shell/pipe_mail" "/bin/mail"
```

Comment:
This code won't link `/bin/mail` if `/usr/bin/mail` is missing. I would suggest the following implementation which links both files independently.
```suggestion
function symlink_binary() {
    local source="$1"
    local target="$1"

    if [ ! -f "$target" ]; then
        echo "Failed to link $target->$source"
        return 1
    fi

    mv "$target" "$target.backup"
    ln -s "$source" "$target"
    echo "Linked $target->$source"
}

symlink_binary "$COMMON_REPO_DIR_MUTUAL_LOC/shell/pipe_mail" "/bin/mail"
symlink_binary "$COMMON_REPO_DIR_MUTUAL_LOC/shell/pipe_mail" "/usr/bin/mail"
```